tests/sys_crypto/tests-crypto-modes-ccm.c

```diff
@@ -1113,7 +1138,7 @@ static void test_decrypt_op(const uint8_t *key, uint8_t key_len,
     len = cipher_decrypt_ccm(&cipher, adata, adata_len,
                              mac_length, len_encoding,
                              nonce, nonce_len, encrypted, encrypted_len, data);
-    TEST_ASSERT_MESSAGE(len > 0, "Decryption failed");
+    TEST_ASSERT_MESSAGE(len >= 0, "Decryption failed");
 
```

Comment:
Can you double check if this is needed? And if so, make sure that this case is documented for `cipher_decrypt_ccm`?

Reply:
This is because if `INPUT_LEN = 0` then the decrypted message would be of length 0, which is not incorrect. In the function documentation it says that on error it returns a negative error code. I can add a message to state that the length can be 0, and make explicit that the `input_length` range start at `0`, is that what you meant?

Reply:
Did just that.